README.md

```diff
@@ -10,7 +10,8 @@ It is primarily focused on FTA which offers possibilities for system partonomy d
 ```shell script
 npm install
 npm run build
-npm run preview
+npm run preview (only preview)
+npm run dev (for development)
```

Comment:
```suggestion
```

Reply:
@Kasmadei 
We have "convention" in our projects to not mix development and production notes together but instead have them in separate sections of README. I will create a separate PR to fix the issue that you wanted to tackle with those proposals.